### DISCUSSION TITLE
Stream Recording Empty Files

### DISCUSSION THREAD

**Discussion by RyanGreeley** [mrlt8/docker-wyze-bridge#543] [Q&A]
Hello,
I've enabled stream recording for my V3 cams. I see that it's creating the video files but they all show up as 0kb. Any thoughts on why this is happening?

```
ryan@ubuntu:~/recordings/DRIVEWAY$ ls -lh
total 0
-rw-r--r-- 1 root root 0 Sep 17 18:46 DRIVEWAY_2022-09-17_18-46-20_EDT.mp4
-rw-r--r-- 1 root root 0 Sep 18 12:20 DRIVEWAY_2022-09-18_12-20-22_EDT.mp4
-rw-r--r-- 1 root root 0 Sep 18 12:44 DRIVEWAY_2022-09-18_12-44-22_EDT.mp4
-rw-r--r-- 1 root root 0 Sep 18 13:22 DRIVEWAY_2022-09-18_13-22-04_EDT.mp4
```

**Comment by mrlt8** [comment]
I believe another use had a similar issue https://github.com/mrlt8/docker-wyze-bridge/discussions/509#discussioncomment-3413735, but it seemed to eventually resolve itself.
